homeassistant/components/ssdp/__init__.py

```diff
@@ -121,7 +122,7 @@ async def _process_entry(self, entry):
             info.update(await info_req)
 
         domains = set()
-        for domain, matchers in SSDP.items():
+        for domain, matchers in self._entries.items():
```

Comment:
Let's rename it so it's clear which entry refers to the "entry matchers" and which are the found entries.
```suggestion
        for domain, matchers in self._integration_matchers.items():
```